content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -33,7 +33,7 @@ folder structure. By default it will create:
 
 ```text
```

Comment:
The scaffolded files has changed. Can you update this section with the information from here? https://www.notion.so/cypressdx/Changes-for-10-0-0-1dca213837b84d6ea297c2d12c615f3a#fadb0af786ac40a5b4788e2e9a5d5c07

You'll want to show End-to-End, Component, and Both, as well as JavaScript vs TypeScript, probably in tabbed interfaces. I think what makes sense would be to have a single `<code-group>` that has a JavaScript and TypeScript tab. In each tab, I believe you can have multiple code blocks, so give that a try. One for each of End-to-End, Component, and Both.

Reply:
What do you think of this presentation? https://github.com/cypress-io/cypress-documentation/pull/4328/commits/3aefca4ae6e19696240d4e7b899ee7f631ef6691